.github/workflows/release.yml

```diff
@@ -93,3 +99,22 @@ jobs:
           echo "scm.url=scm\:git\:git@github.com\:${GITHUB_REPOSITORY}.git" > release.properties
           echo "scm.tag=${{ github.event.inputs.perform_version }}" >> release.properties
           mvn ${MAVEN_FLAGS} release:perform
+      - name: Send Repository Dispatch Event
+        # We not send any repository dispatch if parent_version and perform_version set
+        if: success() && github.event.inputs.send_repository_dispatch == 'yes' && github.event.inputs.parent_version == '' && github.event.inputs.perform_version == ''
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_SECRET }}
+          OWNER: xsalefter
+          REPO: kb-oss-parent
+          # using double quote in request body because we need to parse variable from string
+        run: |
+          VERSION_WITH_SNAPSHOT=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
```

Comment:
I think this is the wrong version. If version is `1.0.0-SNAPSHOT`, after `release:prepare` tags `1.0.0`, it sets the version to `1.0.1-SNAPSHOT`. I suspect you would see `1.0.1` here. Try `git describe --abbrev=0 | cut -d '-' -f 3`